src/components/sbb-wagon/readme.md

```diff
@@ -0,0 +1,68 @@
+# sbb-wagon
+A sbb-wagon represents a train compartment. It can be a locomotive, a blocked passage or a wagon. It is used inside the 
+sbb-sector element. 
+
+**Note:**
+A sbb-wagon of type wagon has the possibilities of slotting icons. They will be applied internally 
+into a list (ul>li) element and requires a title for the list as well as an aria-label for each slotted icon 
+(see example below).
+
+```html
+<sbb-wagon type="locomotive" accessibility-label-wagon="Locomotive" accessibility-additional-wagon-text="Top of the train"></sbb-wagon>
+
+<sbb-wagon type="blocked" accessibility-label-wagon="Passage blocked"></sbb-wagon>
```

Comment:
This is not technically a wagon, but I'm not sure whether the effort to create a `sbb-wagon-blocker` is worth it?